src/lib/stores/data-encoder-config.ts

```diff
@@ -4,7 +4,7 @@ import { persistStore } from '$lib/stores/persist-store';
 import type { DataEncoderStatus } from '$lib/types/global';
 import { has } from '$lib/utilities/has';
 
-export const codecEndpoint = persistStore('endpoint', null, true);
+export const codecEndpoint = persistStore<string>('endpoint', null, true);
```

Comment:
Wondering if we should consider enabling `strictNullChecks`?

Reply:
We probably should just not use `null` here, and use `undefined instead

`export const codecEndpoint = persistStore<string | undefined>('endpoint', undefined, true);`

Although I see `persistStore` is explicitly typed to accept `null` as a default value, I'm not sure why that is.

Reply:
(this is probably fine for now though)

Reply:
Yeah left it `null` for now as I didn't track down if there were any conditionals that relied on it being `null` or something like that. I turned on `strictNullChecks` just to see if we do this elsewhere and there were a bunch of places that popped up.